pkg/resourcegenerator/resourceutils/helpers.go

```diff
@@ -1,9 +1,20 @@
 package resourceutils
 
 import (
+	"regexp"
+	"strings"
+
 	skiperatorv1alpha1 "github.com/kartverket/skiperator/api/v1alpha1"
+	"github.com/kartverket/skiperator/pkg/log"
 	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
-	"strings"
+)
+
+const LabelValueMaxLength = 63
+
+var (
+	logger                  = log.NewLogger().WithName("resourceutils")
```

Comment:
we can pass the logger from the reconciliation object, since we are creating a new logger per reconciliation it might be clearer

Reply:
or just return error and log in the caller